english/marshmallow.md

```diff
@@ -19,6 +19,8 @@ January 2017 approaches. You have two presidential candidates to vote for.
 
 [Click](vote-the-president/hillary/Hillary.md) here to vote for Hillary Clinton.
 
+These options depress you, so you decide not to vote at all, and instead to [write a symphony](write-a-symphony/symphony.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.